packages/rocketchat-api/server/v1/users.js

```diff
@@ -271,6 +271,77 @@ RocketChat.API.v1.addRoute('users.createToken', { authRequired: true }, {
 	}
 });
 
+RocketChat.API.v1.addRoute('users.getPreferences', { authRequired: true }, {
+	get() {
+		const user = this.getUserFromParams();
```

Comment:
You are still allowing to get data from any user you pass in params, you need to use the `this.userId` to find the logged user record

Reply:
Well @rodrigok this is a bit of a chicken and an egg situation, because if you check the file thoroughly, you will find that on 7 other methods we are defining the user in the same way. However, I do agree that we can narrow the scope down and enforce a more secure fashion. 
So I'm changing that now and see if you like it. Thx!